status-im/status-react
# Communicate with bootnodes, mailservers and whisper nodes by :443 (staging, test, beta)

fixes #6645
fixes #6351

### Summary:

Essentially, we use HTTPS port :443 to connect to our nodes.

TODO:
- [x] Test, then deploy :443 mirroring on the `eth.beta` too.

#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- public chats
- group chats
- mailservers

**NB!** "test" or "staging" fleet should be used to test it.


### Steps to test:
- Restrict connections on the network to ports :443 and :80
- Launch Status, switch to the staging fleet.
- Chat should work, the app shouldn't be offline.




status: ready
